### PR TITLE
Fixes logo size and wrapping

### PR DESCRIPTION
**Pull request by AZeghers**
Sometimes the logo would have been cut off or stretched, this should hopefully take care of it.

@ostoc Please review as well

### CODE REVIEW

**Comment by ostoc** [comment]
The open nav bar icon is not centered
 
Screen Shot 2019-05-20 at 14 58 24

**Comment by AZeghers** [comment]
@ostoc Good catch, anything else?

File: packages/kotti-navbar/src/KtNavbarLogo.vue [hunk 1]
```diff
@@ -1,17 +1,36 @@
...
+			<i
+				:class="iconClass"
+				v-text="iconText"
+				@click.stop="$KtNavbar.toggle()"
+			/>
+		</div>
+		<div v-else :class="objectClass" @click="$emit('logoClick')">
+			<div
+				:style="
+					`background-image: url(${themeColor.logoUrl}); height: ${
```
Comment (FlorianWendelborn) [inline packages/kotti-navbar/src/KtNavbarLogo.vue]: This should be moved to a `computed` property.

File: packages/kotti-navbar/src/KtNavbarLogo.vue [hunk 2]
```diff
@@ -1,17 +1,36 @@
...
+						themeColor.logoHeight
+					}`
+				"
+				v-if="themeColor.logoUrl"
```
Comment (FlorianWendelborn) [inline packages/kotti-navbar/src/KtNavbarLogo.vue]: `v-if` should be the first thing, no the second. It’s way more important than some css.

File: packages/kotti-navbar/src/KtNavbarLogo.vue [hunk 3]
```diff
@@ -1,17 +1,36 @@
...
+				@click.stop="$KtNavbar.toggle()"
 			/>
-			<div v-if="!themeColor.logoUrl && !isNarrow" v-text="labelText" />
-			<i class="yoco" v-text="iconText" @click.stop="$KtNavbar.toggle()" />
 		</div>
 		<div class="navbar-logo--mobile">
-			<div v-if="!themeColor.logoUrl" v-text="labelText" />
-			<img :src="themeColor.logoUrl" />
+			<div
+				:style="`background-image: url(${themeColor.logoUrl})`"
```
Comment (FlorianWendelborn) [inline packages/kotti-navbar/src/KtNavbarLogo.vue]: Should be moved to a computed property. Perhaps even the same one a above (you can have multiple `style` attributes).

File: packages/kotti-pagination/src/Pagination.vue [hunk 4]
```diff
@@ -1,17 +1,13 @@
 <template>
 	<div>
 		<ul class="pagination">
-			<li :class="paginatorClasses(1, 'disabled')" @click="previousPage">
```
Comment (FlorianWendelborn) [inline packages/kotti-pagination/src/Pagination.vue]: Are you sure this belongs here?

File: packages/kotti-pagination/src/Pagination.vue [hunk 5]
```diff
@@ -21,28 +17,52 @@
 </template>
 
 <script>
+import ContractedPaginator from './components/ContractedPaginator.vue'
```
Comment (FlorianWendelborn) [inline packages/kotti-pagination/src/Pagination.vue]: Are you sure this belongs here?

File: packages/kotti-pagination/src/components/ContractedPaginator.vue [hunk 6]
```diff
@@ -0,0 +1,94 @@
+<template>
```
Comment (FlorianWendelborn) [inline packages/kotti-pagination/src/components/ContractedPaginator.vue]: Are you sure this belongs here?

File: packages/kotti-pagination/src/components/ExpandedPaginator.vue [hunk 7]
```diff
@@ -0,0 +1,47 @@
+<template>
```
Comment (FlorianWendelborn) [inline packages/kotti-pagination/src/components/ExpandedPaginator.vue]: Are you sure this belongs here?

File: packages/kotti-pagination/src/components/FractionatedPaginator.vue [hunk 8]
```diff
@@ -0,0 +1,20 @@
+<template>
```
Comment (FlorianWendelborn) [inline packages/kotti-pagination/src/components/FractionatedPaginator.vue]: Are you sure this belongs here?

File: packages/kotti-style/utilities/_cursors.scss [hunk 9]
```diff
@@ -15,4 +15,7 @@ $cursors: (
 	.#{$element} {
 		cursor: $cursor;
 	}
+	.#{$element}-important {
```
Comment (FlorianWendelborn) [inline packages/kotti-style/utilities/_cursors.scss]: Are you sure this belongs here?

**Pull request merged**